### ISSUE TITLE
Consider changing SocksiPy-branch requirement to pysocks

### ISSUE THREAD

**Issue by nirik** [ProgVal/Limnoria#1246]
My understanding is that SocksiPy-branch is pretty dead, there's no upstream googlecode repo anymore and it's not been updated in about 6 years. 
Someone forked it as https://github.com/brendoncrawford/socksipy-x which in turn was forked as https://github.com/Anorov/PySocks

Hopefully it's stable there now and PySocks could be used instead of SocksiPy-branch?

I'm not sure all what might need adjusting, it may just be a drop in replacement. 

Thanks for the great project. :)

**Comment by br00tal** [comment]
I was just going to open a ticket for this, but this one already exists.  I can confirm that `SocksiPy-branch` doesn't really work (at least not with any proxies I tried), and `PySocks` worked for me as a drop-in replacement.  All I needed to do was replace `SocksiPy-branch` with `PySocks` in `requirements.txt` and them use `pip` to install all of them.

Right now my `Dockerfile` has to replace that line in `requirements.txt` prior to installing Limnoria. It'd be nice to simply have that file updated so everything works out-of-the-box.
